### ISSUE TITLE
Why not use python3?Why?

### ISSUE THREAD

**Issue by wangwei1998** [jofpin/trape#171]
As a beginner，I  feel interested in your project and appreciate for your hard work.  it takes me almost one day to download py2 and other things,however,i failed .I sincerely hope you use py3 which is much friendly to us.THANK YOU! Looking forward to your V3
![IMG_0797(20190716-184933)](https://user-images.githubusercontent.com/52748718/61298913-a94fcb00-a811-11e9-82c6-37505494f20e.jpg)
.0!

**Comment by hwac121** [comment]
I have py1,2, and 3 installed and this works fine on Parrot Security.

**Comment by wangwei1998** [comment]
> I have py1,2, and 3 installed and this works fine on Parrot Security.

Can you help  me to solve this problem?

**Comment by hwac121** [comment]
I am happy to try, what problem are you having exactly?

**Comment by hwac121** [comment]
1. You may want to run: python3 -m pip install --upgrade pip
2. Then cd to the setup directory
3. python3 -m pip install --no-cache-dir -r requirements.txt

**Comment by wangwei1998** [comment]
> 1. You may want to run: python3 -m pip install --upgrade pip
> 2. Then cd to the setup directory
> 3. python3 -m pip install --no-cache-dir -r requirements.txt After that just: python3 social_mapper.py -h

![捕获](https://user-images.githubusercontent.com/52748718/61697263-75bcf580-ad69-11e9-8a38-91117aceb676.PNG)
it says that No module named 'httplib'

**Comment by hwac121** [comment]
yeah glad you understood that, I was working with social mapper when I wrote that LOL.
let me take another look at it...

**Comment by wangwei1998** [comment]
> ## Sorry bout that...
> yeah glad you understood that, I was working with social mapper when I wrote that LOL.
> let me take another look at it...
> 
> Ok, so do this first: python2 -m pip install -r requirements.txt
> Then try python2 trape.py -h
i solved that question which used py.but now,i faced new problem
![捕获](https://user-images.githubusercontent.com/52748718/61811965-2ebc2680-ae75-11e9-8326-1f7b62b5b26f.PNG)can you help me?
